ratatool-sampling/src/test/scala/com/spotify/ratatool/io/FixRandomData.scala

```diff
@@ -20,7 +20,7 @@ package com.spotify.ratatool.io
 import com.spotify.ratatool.avro.specific.TestRecord
 
 import java.util.{HashMap => JHashMap, Map => JMap}
-import scala.collection.JavaConversions._
+import scala.jdk.CollectionConverters._
```

Comment:
Should we migrate all instance of `scala.collection.JavaConverters` to this as well? That's the deprecated one now right?